Configurations.md

```diff
@@ -2294,6 +2294,39 @@ use std::sync::Arc;
 use uuid::Uuid;
 ```
 
+#### Example for custom groups:
+
+Discard existing import groups, and create groups as specified by the wildcarded list.
+Handy aliases are supported:
+
+- `$std` prefix is an alias for standard library (i.e `std`, `core`, `alloc`);
+- `$crate` prefix is an alias for crate-local modules (i.e `self`, `crate`, `super`).
+
+For a given config:
+
+```toml
+group_imports = [
+    ["$std::*", "proc_macro::*"],
+    ["my_crate::*", "crate::*::xyz"],
+    ["$crate::*"],
+]
+```
+
+The following order would be set:
+
+```rust
```

Comment:
Not really sure how to deal with this one. I could just remove rust syntax so the config test passes, or perhaps there's a better option?

Reply:
I think you'll need to implement `FromStr` for `GroupImportsTactic`. You can take a look at the config documentation added by  https://github.com/rust-lang/rustfmt/pull/5347, as an example.

Reply:
@l4l Will you be able to make these changes or should I create a new PR?

Reply:
Was having a vacation with no internet. Hopefully will update the PR within this weekend